### PR TITLE
Deployments

### PR DESCRIPTION
**Pull request by svidhani**
### Requirements

Refine deployments

### CODE REVIEW

File: core/jazz_deployments-event-handler/test/test.js [hunk 1]
```diff
@@ -622,8 +622,8 @@ describe("updateDeployments", () => {
     });
     index.updateDeployments(JSON.stringify(res), deploymentPayload, configData, "temp_auth").then((obj) => {
       sinon.assert.calledOnce(processRequestStub);
-      processRequestStub.restore();
     })
+    processRequestStub.restore();
```
Comment (pspani) [inline core/jazz_deployments-event-handler/test/test.js]: why was this moved out from the callback

Comment (supritat) [reply core/jazz_deployments-event-handler/test/test.js]: updated
